<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Растеризация</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>780</width>
         <height>410</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="maximumSize">
             <size>
              <width>15</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>x1</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBox">
              <property name="minimum">
               <double>-100.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="horizontalSlider">
              <property name="maximum">
               <number>10</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="minimumSize">
             <size>
              <width>6</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>15</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>1</width>
              <height>0</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>6</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>y1</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
              <property name="minimum">
               <double>-100.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="horizontalSlider_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="maximumSize">
             <size>
              <width>15</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>x2</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
              <property name="minimum">
               <double>-100.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="horizontalSlider_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>15</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>y2</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
              <property name="minimum">
               <double>-100.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="horizontalSlider_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>Пошаговый алгоритм</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Алгоритм ЦДА</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Алгоритм Брезенхема </string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Алгоритм Брезенхема (окружность)</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Scale Up</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Scale Down</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="a_StepByStep">
   <property name="text">
    <string>Пошаговый</string>
   </property>
  </action>
  <action name="a_Bresenham">
   <property name="text">
    <string> Брезенхэма</string>
   </property>
  </action>
  <action name="a_DDA">
   <property name="text">
    <string>ЦДА</string>
   </property>
  </action>
  <action name="a_circle">
   <property name="text">
    <string>Брезенхэма(окружность)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
